### PR TITLE
VD for Animal's list

### PR DESCRIPTION
**Pull request by kobsonaut**

### CODE REVIEW

File: TOZ_iOS/ListOfAnimalsArrayResponseMapper.swift [hunk 1]
```diff
@@ -26,7 +26,8 @@ final class ListOfAnimalsArrayMapper: ArrayResponseMapper<AnimalItem>, ResponseM
             guard let imageString = jsonNode["imageUrl"] as? String? else { throw ResponseMapperError.responseParsingFailed }
             var imageURL: URL? = nil
             if let imageString = imageString {
-                imageURL = BackendConfiguration.shared.baseURL.appendingPathComponent(imageString)
+//                imageURL = BackendConfiguration.shared.baseURL.appendingPathComponent(imageString)
+                imageURL = URL(string: imageString)
```
Comment (filip-zielinski) [inline TOZ_iOS/ListOfAnimalsArrayResponseMapper.swift]: Is this a temporary change just to show images of animals when using mocked backend? I believe we talked on the last meeting that we're going to use images URLs that are relative to backend, even though it doesn't return correct endpoints at the moment.

Comment (kobsonaut) [inline TOZ_iOS/ListOfAnimalsArrayResponseMapper.swift]: Yes. It's for mocked backend. We can keep this line as a comment, but it's needed. :)

Comment (kobsonaut) [inline TOZ_iOS/ListOfAnimalsArrayResponseMapper.swift]: I've added also a fix for displaying the view on different devices. Thanks for review.

**Comment by kobsonaut** [comment]
All issues has been fixed.

**Comment by piotr-tobolski** [comment]
You forgot about the unit tests

**Comment by kobsonaut** [comment]
Done :)

**Pull request merged**